app/manifest/v3/_base.json

```diff
@@ -47,6 +47,7 @@
   ],
   "default_locale": "en",
   "description": "__MSG_appDescription__",
+  "host_permissions": ["file://*/*", "http://*/*", "https://*/*"],
```

Comment:
Why was `file://` included here? I didn't think we communicated with pages that use that scheme.

Reply:
I was thinking for local dev but people can serve a page locally so we can remove this.

Reply:
Ah OK. I don't mind it being here, I was just curious.

I was thinking this might make it break when viewing non-webpage files :thinking: But maybe not, I didn't test it.

Reply:
file:// is a bit unpredictable, we could consider removing it in production build or entirely. Unless there's usecases that depend on it.

While we're at it - http should only be allowed for localhost IMHO - on a http site someone can get access to your window.ethereum with a simple MITM

Reply:
>I don't mind it being here, I was just curious.

We should at the very least be consistent about whether we're supporting `file://` for injection or not. I see that it was removed from the `registerContentScripts` call, but it remains here in `host_permissions`. Seems like it should be included or removed from both places.

Reply:
Updated and I agree.  I removed `file://` since I can't imagine it used often -- we can add back if asked to reconsider.

Reply:
I'm only just now realizing that we've been injecting contentscripts into `file://` pages since at least 2016. Sorry about that :facepalm:  I was confused, and thought it was being introduced here.

I've pinged Dan Finlay in Slack to see if he remembers why it was added. It didn't seem clear from context. It was definitely done on purpose at some point though, so maybe it would be best to leave it until we find out the reason.

Reply:
It look like this inconsistency remains. `file://` is listed here in `host_permissions`, but the contentscript doesn't get injected into `file://` URLs.